### DISCUSSION TITLE
No way to disable or customize keyboard shortcut?

### DISCUSSION THREAD

**Discussion by sjml** [aclap-dev/video-downloadhelper#725] [Help and Q&A]
### Your question

I do a lot of text editing in a web app, and so frequently use Cmd-Shift-UpArrow to select everything from my cursor to the beginning of text field. Now, though, this pops up the VideoDownloadHelper pane. I use VDH occasionally and usually invoke it from the menu when I do... it's definitely not a bad idea to have a keyboard shortcut, but we should be able to turn it off or change it. For now I've disabled the plugin, but is there any other way to keep it from responding to that shortcut?  

### Addon & CoApp version and details

version: 9.0.1.13
target: mozilla
channel: stable
lang: en-US
coapp: {"found":false,"error":"No such native application net.downloadhelper.coapp"}
license: {"unset":true}
platform: aarch64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:126.0) Gecko/20100101 Firefox/126.0
{
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "open_count_store": 1
}

**Comment by GrampaWildWilly** [comment]
![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/4c27856e-0005-4a9f-95d2-45c7bc90b21c)
![#02](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/e2918220-7b29-497d-9dac-b2028f61d5d3)
![#03](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/f11eb3df-6205-4fac-aa07-b8866d7c17e3)

**Reply by sjml** [reply]
Great, thanks! I was looking all over the extension's own settings, not realizing it was browser-level.

**Comment by koppelaar** [comment]
The same can be done in Chrome here: chrome://extensions/shortcuts
